app/code/Magento/Backend/Test/Unit/Service/V1/ModuleServiceTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Backend\Test\Unit\Model;
```

Comment:
Please, adjust the namespace to correspond the file path

Reply:
@rogyar thank you!
I've updated  the namespace.